Formula/smpq.rb

```diff
@@ -0,0 +1,73 @@
+class Smpq < Formula
+  desc "StormLib MPQ archiving utility"
+  homepage "https://launchpad.net/smpq"
+  url "https://launchpad.net/smpq/trunk/1.6/+download/smpq_1.6.orig.tar.gz"
+  sha256 "b5d2dc8a5de8629b71ee5d3612b6e84d88418b86c5cd39ba315e9eb0462f18cb"
+  license "GPL-3.0-or-later"
+
+  livecheck do
+    url :homepage
+    regex(/href=.*?smpq[._-](\d+(?:\.\d+)+)[._-]orig/i)
+  end
+
+  depends_on "cmake" => :build
+  depends_on "stormlib"
+
+  # Patch to allow compilation on macOS
+  patch :p1, <<~EOS
```

Comment:
>     * Formulae should not require patches to build. Patches should be submitted and accepted upstream first.

Reply:
Looks like no release since 2016, is this still maintained?

Reply:
I'm unsure, to be honest.  I do have a bug open with the project on Launchpad with a link to a patch to do what I'm doing in this patch, but I don't know whether it will be accepted or not.